e2e-tests/haproxy/conf/some-name.yml

```diff
@@ -0,0 +1,99 @@
+apiVersion: pxc.percona.com/v1
```

Comment:
please rename it to `haproxy` (similar to other tests)
reason: each unique configuration has its own name